tajo-algebra/src/main/java/org/apache/tajo/algebra/WindowSpec.java

```diff
@@ -128,12 +128,14 @@ public boolean equals(Object obj) {
   public static enum WindowFrameStartBoundType {
     UNBOUNDED_PRECEDING,
     CURRENT_ROW,
-    PRECEDING
+    PRECEDING,
+    FOLLOWING
   }
```

Comment:
Hmm... I think the discussion that you linked is not complete one
because UNBOUNDED_PRECEDING is only available at start bound type while UNBOUNDED_FOLLOWING is only available as an end bound type.
I tested in psql as following
```
mydb=# select id, max(value) over (partition by id rows between unbounded following and unbounded following) from test3;
ERROR:  frame start cannot be UNBOUNDED FOLLOWING
LINE 1: ...id, max(value) over (partition by id rows between unbounded ...
                                                             ^
```
and saw the error as I expected.

So, I think it needs separation between start bound type and end bound type.